docs/standard/modelling/dates-guidance.rst

```diff
@@ -65,7 +65,7 @@ The new statement:
 * has the same ``statementDate``, because the date of the original information submission has not changed
 * has a new ``publicationDate``
 * has a corrected ``interest.startDate``
-* has an Annotation explaining the correction
+* has an 'Annotation <https://standard.openownership.org/en/main/standard/reference.html#schema-annotation>'_ explaining the correction
```

Comment:
```suggestion
* has an :ref:`Annotation <schema-annotation>` explaining the correction
```

This is a slightly more robust way of linking within the docs. It links to an anchor in the Schema reference page ('.. _schema-annotation:')